cranelift/codegen/src/isa/x64/pcc.rs

```diff
@@ -165,11 +165,6 @@ pub(crate) fn check(
             }
         }
 
-        Inst::Mov64MR { ref src, dst } => {
-            check_load(ctx, Some(dst.to_writable_reg()), src, vcode, I64, 64)?;
-            Ok(())
-        }
-
```

Comment:
@cfallin: I expect 3 PCC tests to fail due to this. I tried to look at how I might re-write this based on the assembler but at this point we only have assembler types to deal with. What to do?

Reply:
It's fine to remove the newly failing tests for now, I think -- PCC shouldn't get in the way of progress here (I'm happy to update it later when I next get a timeslice for it). Feel free to remove the test directives in the files and leave a note instead "disabled until PCC is migrated to understand new assembler instructions" or similar...